<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatDispoit</class>
 <widget class="QWidget" name="CreatDispoit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="login_passwdLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>101</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_passwdInput">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>91</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="login_repeatPasswdState">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>171</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="login_addressLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>地址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_addressInput">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_nameInput">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>51</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="login_confirmPasswdLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>141</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_accountInput">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>11</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777211</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_confirmPasswdInput">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>131</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="login_nameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>61</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="login_accountLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>21</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>账号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_interestInput">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="text">
    <string>1.98</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="login_typeCombobox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>一年期</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>三年期</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>五年期</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="login_typeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>储种</string>
   </property>
  </widget>
  <widget class="QLabel" name="login_interestLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>利息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_okButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>540</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_backButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>540</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_employeename">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="login_employeenameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>员工姓名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_employeeInput">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>410</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="login_employeeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>员工号</string>
   </property>
  </widget>
  <widget class="QLabel" name="login_principalLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>本金</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_principal">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
